README.md

```diff
@@ -403,6 +444,231 @@ You can use `urlEndpoint` prop in an individual `IKImage` to change url for that
+| loading  | String |Optional. Pass `lazy` to lazy load videos. Note: Component does not accept change in this value after it has mounted. |
+| lqip  | Object |Optional. You can use this to show a low-quality blurred placeholder while the original video is being loaded e.g. `{active:true, quality: 20, blur: 6, raw: "n-lqip_named_transformation"`}. The default value of `quality` is `20`, and `blur` is `6`. If `raw` transformation is provided, SDK uses that and ignores the `quality` and `blur` parameters. <br /> Note: Component does not accept change in this value after it has mounted.|
+
+### Basic resizing examples
+
+```js
+<IKContext urlEndpoint="https://ik.imagekit.io/your_imagekit_id">
+  // Video from related file path with no transformations - https://ik.imagekit.io/your_imagekit_id/default-video.mp4
+  <IKVideo
+    path="/default-video.mp4"
+  />
+
+  // Video resizing - https://ik.imagekit.io/your_imagekit_id/tr:w-h-300,w-400/default-video.mp4
+  <IKVideo
+    path="/default-video.mp4"
+    transformation={[{
+      height:300,
+      width:400
+    }]}
+  />
+
+  // Loading video from an absolute file path with no transformations - https://www.custom-domain.com/default-video.mp4
+  <IKVideo
+    src="https://www.custom-domain.com/default-video.mp4"
+  />
+
+  // Using a new tranformation parameter which is not there in this SDK yet - https://ik.imagekit.io/your_imagekit_id/tr:custom-value/default-video.mp4
+  <IKVideo
+    path="/default-video.mp4"
+    transformation={[{
+      custom: 'value'
+    }]}
+  />
+</IKContext>
+```
+
+The `transformation` prop is an array of objects. Each object can have the following properties.
+
+```js
+// It means first resize the video to 400x400 and then rotate 90 degree
+transformation = [
+  {
+    height: 400,
+    width: 400,
+    rt: 90
+  }
+]
+```
+
+See the complete list of transformations supported in ImageKit [here](https://docs.imagekit.io/features/video-transformation). The SDK gives a name to each transformation parameter e.g. `height` for `h` and `width` for `w` parameter. It makes your code more readable. If the property does not match any of the following supported options, it is added as it is.
+
+### List of supported transformations
+<details>
+<summary>Expand</summary>
+
+| Supported Transformation Name | Translates to parameter |
+|-------------------------------|-------------------------|
+| height | h |
+| width | w |
+| aspectRatio | ar |
+| quality | q |
+| crop | c |
+| cropMode | cm |
+| focus | fo |
+| format | f |
+| radius | r |
+| background | bg |
+| border | b |
+| rotation | rt |
+| blur | bl |
+| named | n |
+| trimmingStartOffset | so |
+| trimmingEndOffset | eo |
+| trimmingDuration | du |
+| layering | l |
+| layerInfo | i |
+| layerXPosition | lx |
```

Comment:
This is just a random change only in Readme.md and no where else
<img width="1096" alt="image" src="https://user-images.githubusercontent.com/45416977/185405303-9b39b3dd-c0ba-4b4c-9e26-ce6b41f7908a.png">

This would not work because the underlying JS library doesn't understand `layerXPosition` parameter at all.

This table should match that of JS SDK 1.5.2. version.

Reply:
Fixed.